proto/tenancy_config/active_kce_configuration.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package holms.types.tenancy_config;
+option csharp_namespace = "HOLMS.Types.TenancyConfig";
+option go_package = "tenancyconfig";
+
+enum ActiveKCEConfiguration {
```

Comment:
Can you be a little more explicit here and less "jargony"? Just call this "KeyCardBrand" or "ActiveKeyCardBrand".

Reply:
Change to ActiveKeySystem